sherpa/fit.py

```diff
@@ -1132,16 +1131,15 @@ def calc_chisqr(self):
 
         Returns
         -------
-        chisq : array or `None`
+        chisq : array or ``None``
            The chi-square value for each bin of the data, using the
-           current statistic (as set by `set_stat`).  A value of
-           `None` is returned if the statistic is not a chi-square
+           current statistic (as set by `sherpa.astro.ui.set_statset_stat`).
+           A value of ``None`` is returned if the statistic is not a chi-square
```

Comment:
```suggestion
           current statistic. A value of ``None`` is returned if the
           statistic is not a chi-square
```

So, this is at a much "lower" level then sherpa.astro.ui/sherpa.ui, so my  original inclusion of the text mentioning `set_stat` is wrong and should be removed (which also fixes the `set_statset_stat` duplication here).